GameOn-Frontend/app/(auth)/sign-in.tsx

```diff
@@ -0,0 +1,226 @@
+              } finally {
+                setSubmitting(false);
+              }
+            }}
+          >
+            {({
+              values,
+              errors,
+              touched,
+              handleBlur,
+              handleSubmit,
+              isSubmitting,
+              status,
+              setStatus,
+              setFieldValue,
+            }) => (
+              <>
+                
+                <View style={{ gap: 16 }}>
+                  <LabeledInput
+                    label="Email"
+                    placeholder="example@example.com"
+                    value={values.email}
+                    onChangeText={(t: string) => {
+                      setStatus(undefined);        
+                      setFieldValue('email', t);
+                    }}
+                    onBlur={handleBlur('email')}
+                    keyboardType="email-address"
+                    autoCapitalize="none"
+                    error={touched.email && errors.email ? errors.email : undefined}
+                  />
+
+                  <LabeledInput
+                    label="Password"
+                    placeholder="**********"
+                    value={values.pwd}
+                    onChangeText={(t: string) => {
+                      setStatus(undefined);        
+                      setFieldValue('pwd', t);
+                    }}
+                    onBlur={handleBlur('pwd')}
+                    secureTextEntry={!showPwd}
+                    rightIcon={
+                      <Pressable onPress={() => setShowPwd((s) => !s)} hitSlop={8}>
+                        <Ionicons name={showPwd ? 'eye-off-outline' : 'eye-outline'} size={20} color="#000000" />
+                      </Pressable>
+                    }
+                    error={touched.pwd && errors.pwd ? errors.pwd : undefined}
+                  />
+
+                  <Pressable onPress={() => { /* TODO: forgot password flow */ }} style={styles.forgotWrap}>
+                    <Text style={styles.forgotText}>Forgot Password</Text>
+                  </Pressable>
+                </View>
+
+                {status ? (
+                  <View style={styles.statusBox}>
+                    <Text style={styles.statusText}>{status}</Text>
+                  </View>
+                ) : null}
+
+                <Pressable
+                  onPress={() => handleSubmit()}
+                  disabled={isSubmitting}
+                  style={({ pressed }) => [
+                    styles.cta,
+                    (pressed || isSubmitting) && { opacity: 0.85 },
+                    isSubmitting && { opacity: 0.6 },
+                  ]}
+                >
+                  {isSubmitting ? <ActivityIndicator /> : <Text style={styles.ctaText}>Log In</Text>}
+                </Pressable>
+
+                <View style={{ marginTop: 'auto' }}>
+                  <Text style={styles.metaText}>
+                    Donâ€™t have an account?{' '}
```

Comment:
```suggestion
                    Donâ€™t have an account?{' '}
```
Small mishap over here

Reply:
Please use `&apos;` insead of the apostrophe